solidity/ecdsa/test/WalletRegistry.WalletCreation.test.ts

```diff
@@ -1869,14 +1874,14 @@ describe("WalletRegistry - Wallet Creation", async () => {
               ).to.be.equal(hashUint32Array(expectedMembers))
             })
 
-            // it("should ban misbehaved operators from sortition pool rewards", async () => {
-            //   const now = await helpers.time.lastBlockTime()
-            //   const expectedUntil = now + params.sortitionPoolRewardsBanDuration
+            it("should ban misbehaved operators from sortition pool rewards", async () => {
+              const now = await helpers.time.lastBlockTime()
+              const expectedUntil = now + params.sortitionPoolRewardsBanDuration
 
-            //   await expect(tx)
-            //     .to.emit(sortitionPool, "IneligibleForRewards")
-            //     .withArgs(misbehavedIds, expectedUntil)
-            // })
+              await expect(tx)
+                .to.emit(sortitionPool, "IneligibleForRewards")
```

Comment:
What do you think about adding a test when there are no misbehaved members and the event was not emitted?

Reply:
Added in https://github.com/keep-network/keep-core/pull/2868/commits/6f96d7d22d08da1df35d67ff4a415b2f3c2380be. I attached to the existing `context` to do not introduce one more nesting level.